sys/net/gnrc/netif/ieee802154/gnrc_netif_ieee802154.c

```diff
@@ -129,6 +129,9 @@ static gnrc_pktsnip_t *_recv(gnrc_netif_t *netif)
             gnrc_netif_hdr_t *hdr = netif_snip->data;
             hdr->lqi = rx_info.lqi;
             hdr->rssi = rx_info.rssi;
+            if (rx_info.flags & NETDEV_RX_IEEE802154_INFO_FLAG_TIMESTAMP) {
```

Comment:
Now this needs to be wrapped inside
```suggestion
#if IS_USED(MODULE_GNRC_NETIF_TIMESTAMP)
            if (rx_info.flags & NETDEV_RX_IEEE802154_INFO_FLAG_TIMESTAMP) {
```
as the struct element is not always available.

Reply:
Oh I oversaw that, thanks. But it's `MODULE_SOCK_AUX_TIMESTAP`, right?

Reply:
yea that should also be fine - just thought inside GNRC we might as well check for the GNRC module